### PR TITLE
nic105

### PR DESCRIPTION
**Pull request by petele**
New in Chrome 105 post

### CODE REVIEW

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: site/en/blog/new-in-chrome-105/index.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+Container queries, one of the most highly requested features are landing in
+Chrome 105. They enable developers to query a parent selector for its size and
+styling information, making it possible for a child element to own its
+responsive styling logic, no matter where it lives on a page.
+
+{% Columns %}
+{% Column %}
+They’re similar to a @media query, except they evaluate against the size of
+a container  instead of the size of the viewport.
+{% endColumn %}
```
[the remark this answers is not in the document]
Comment (petele) [reply site/en/blog/new-in-chrome-105/index.md:42]: I want it side-by-side, it fits rather nicely.

File: site/en/blog/new-in-chrome-105/index.md [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+cross-site scripting vulnerabilities by building sanitization into the platform.
+
+To use it, create a new instance of Sanitizer. Then, call `setHTML()` on the
+element you want to insert the sanitized content into.
+
+```js
+const mySanitizer = new Sanitizer();
+const user_input = `<img src="" onerror=alert(0)>`;
+elem.setHTML(user_input, { sanitizer: mySanitizer });
+```
```
[the remark this answers is not in the document]
Comment (jpmedley) [reply site/en/blog/new-in-chrome-105/index.md:124]: It came back to me. Checked the article again. It's fine.

File: site/en/blog/new-in-chrome-105/index.md [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+{% Aside 'warning' %}
+This deprecation will currently only affect sites that use WebSQL via
+`http://`, for example, some intranet or enterprise scenarios. It will
+**not** affect sites that use WebSQL via `https://`.
+{% endAside %}
+
+If you’re using WebSQL in non-secure contexts, you should migrate to IndexDB,
+or another local storage container as soon as possible. For more information
+about this deprecation and the planned deprecation of WebSQL see
+[TODO: post title][dcc-websql-dep].
```
[the remark this answers is not in the document]
Comment (jpmedley) [reply site/en/blog/new-in-chrome-105/index.md]: Ping me when this is done.

File: site/en/blog/new-in-chrome-105/index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (jpmedley) [reply site/en/blog/new-in-chrome-105/index.md:42]: I'm approving this, but I shouldn't. In addition to being a problem for narrow windows, it also violates a logical extension of a writing best practice: "prefer the standard to the offbeat." I encourage you to submit a correction.

**Pull request merged**